packages/commonwealth/client/scripts/views/pages/NotificationSettingsOld/index.tsx

```diff
@@ -203,6 +167,7 @@ const NotificationSettingsPage = () => {
                       disabled={!emailValidated}
                       onClick={() => {
                         try {
+                          // eslint-disable-next-line @typescript-eslint/no-floating-promises
```

Comment:
The lint disables in this file are easily fixable, let's try not to disable link rules where it's unnecessary.